feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -0,0 +1,64 @@
+package com.revenuecat.purchases.google
+
+import com.android.billingclient.api.ProductDetails
+import com.android.billingclient.api.SkuDetails
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.PricingPhase
+import com.revenuecat.purchases.models.StoreProduct
+import org.json.JSONObject
+
+/*fun SkuDetails.toStoreProduct() =
+    StoreProduct(
+        sku,
+        type.toRevenueCatProductType(),
+        price,
+        priceAmountMicros,
+        priceCurrencyCode,
+        originalPrice,
+        originalPriceAmountMicros,
+        title,
+        description,
+        subscriptionPeriod.takeIf { it.isNotBlank() },
+        freeTrialPeriod.takeIf { it.isNotBlank() },
+        introductoryPrice.takeIf { it.isNotBlank() },
+        introductoryPriceAmountMicros,
+        introductoryPricePeriod.takeIf { it.isNotBlank() },
+        introductoryPriceCycles,
+        iconUrl,
+        JSONObject(originalJson),
+    )*/
+
+fun ProductDetails.PricingPhases.readable(): String {
+    return "[" + pricingPhaseList.map { (if (it.billingCycleCount > 1) (it.billingCycleCount.toString() + "*") else "") + it.billingPeriod + "@" + it.formattedPrice }
+        .joinToString() + "]"
+}
+
+fun ProductDetails.toStoreProduct(offerToken: String, pricingPhases: ProductDetails.PricingPhases, offerTags: List<String>) =
+    StoreProduct(
+        productId,
+        ProductType.SUBS,
+        pricingPhases.readable(),
+        100,
+        "USD",
+        "originalPrice",
+        100,
+        title,
+        offerTags.toString(),
```

Comment:
Hmm I think we should leave the tags as an array. Any reason not to? (This might be WIP and that's ok)

Reply:
Oh yeah definitely - this again was just trying to provide some info in purchases_sample without having to change the whole app.